src/shared/locales/en/minisry.json

```diff
@@ -1 +1,67 @@
-
+{
+  "tr_addServiceTime": "Add service time",
+  "tr_hours": "Hours",
+  "tr_bibleStudies": "Individual Bible studies",
+  "tr_btnAdd": "Add",
+  "tr_creditHours": "Credit hours",
+  "tr_addNewStudy": "Add new Bible study",
+  "tr_pioneerSchool": "Pioneer school",
+  "tr_сreditHours": "{{ ministry time }} hours",
+  "tr_SKE": "School for Kingdom Evangelizers",
+  "tr_languageCourse": "Language course",
+  "tr_personalDay": "Personal day",
+  "tr_selectDate": "Select date",
+  "tr_saveChanges": "Save changes",
+  "tr_deleteStudy": "Delete the study",
+  "tr_timeInService": "Time already in service",
+  "tr_timeInServiceDesc": "The time will be added to the ministry timer",
+  "tr_monthlyReport": "Monthly report",
+  "tr_sharedMinistry": "Participated in any form of ministry during this month",
+  "tr_dailyHistory": "Daily history",
+  "tr_btnAddRecord": "Add record",
+  "tr_addRecordDesc": "You don’t have any daily service records yet. Add one here to fill out your monthly report.",
+  "tr_btnSubmitReport": "Submit the report",
+  "tr_badgeGoalHours": "out of {{ministry time}}h",
+  "tr_btnAddCreditHours": "Add hours credit",
+  "tr_extraTime": "You have extra {{ ministry time }} minutes",
```

Comment:
@mrtavor: would it be possible to change this placeholder not to use any space. i.e. `ministryTime`. Thanks